src/pages/ReportDetailsPage.tsx

```diff
@@ -660,6 +656,7 @@ function ReportDetailsPage({policies, report, route, reportMetadata}: ReportDeta
 
         return items;
     }, [
+        beginPDFExport,
```

Comment:
Is adding this intentional or is it a merge conflict issue?

Reply:
It seems needed there, so all good 👍

Reply:
This is here twice, argh - https://github.com/Expensify/App/issues/60964